src/components/OtherHero/index.js

```diff
@@ -30,15 +32,19 @@ const useStyles = makeStyles(({ palette, typography, breakpoints }) => ({
       paddingTop: typography.pxToRem(129.94),
     },
   },
-  overline: {},
+  overline: {
+    position: "relative",
+  },
```

Comment:
will these conflict with the other PR for Other hero?

Reply:
Yap. @KobbyMmo ....but I much prefer for this to get merged first. Then pull the changes to the OtherHero PR....its more manageable like that